pkg/apps/strategy/rolling/rolling_updater.go

```diff
@@ -0,0 +1,640 @@
+/*
```

Comment:
Why do we want to embed it while that's still part of the upstream? As deprecated it'll be there at least for a year so we can avoid maintaining a copy of that throughout that time.

Reply:
@tnozicka i think it is better to have it here, embedded than pulling entire kubectl with this... it reduces the binary size. Also this version works with external client, I guess we will have to switch upstream to external as well, but I dunno the side-effects of doing that. We should definitely look at this in 1.12 kube, but I don't want to block moving deployer out and stripping kubectl from it here.

Reply:
it's fine than